src/applications/gi/selectors/calculator.js

```diff
@@ -443,6 +443,8 @@ const getDerivedValues = createSelector(
 
     const totalHousingAllowance = monthlyRateFinal * termLength;
 
+    const bah = inputs.beneficiaryLocationBah || institution.bah;
```

Comment:
Need to confirm that this is actually the change to the formula needed. There's a lot of references to housing allowance in this that aren't derived from the Basic Allowance for Housing.

Reply:
Basically whenever the input is in an invalid state, this value will be null and it will fall back to the institution bah.